specifications/network/network-address.md

```diff
@@ -92,6 +84,17 @@ NetworkAddress([
 ]) => "/ip4/127.0.0.1/tcp/6080/ln-noise-ik/080e287879c918794170e258bfaddd75acac5b3e350419044655e4983a487120/ln-handshake/0"
 ```
 
+A `NetworkAddress` as a concatenation of `Protocol` segments must have the following characteristics:
+1. The address must contain a Layer3 protocol (e.g. `Ip4` or `Dns`).
+2. The address must contain a Layer4 protocol (e.g. `Tcp` or `Memory`).
+3. A protocol may be used at most once in an address.
+4. The address must not end in a forward slash `/`
+
+Example possible combinations:
+* `/ip4/127.0.0.1/tcp/6080`
+* `/ip4/127.0.0.1/memory/6080`
```

Comment:
should just be `/memory/6080` since `Memory` is ~ L3+L4

Reply:
Yeah, that makes sense